server/models/Schedule.js

```diff
@@ -0,0 +1,21 @@
+
+const mongoose = require("mongoose");
+const Period = require("./Period");
+
+const scheduleSchema = new mongoose.Schema({
+  active: {
+    type: Boolean,
+    required: true
+  },
+  periods: {
+    monday: [Period],
```

Comment:
Not sure about making this an array of periods, it should just be the object. Can we just embed the start and end time too? Seems unnecessary to reference periods. It's a collection we don't need in the database.